clojure/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo -e "Building Clojure Implementations...\n"
+
+##################
+# ONYX
+
+cd onyx-impl
```

Comment:
@punit-naik - lets keep the top-level folder name as the platform name. So, change this to ``onyx``.